frontend/app/src/comps/EarnPositionSummary/EarnPositionSummaryBase.tsx

```diff
@@ -186,10 +189,12 @@ function OpenLink({
   active,
   path,
   title,
+  external,
 }: {
   active: boolean;
   path: string;
   title: string;
+  external?: boolean;
```

Comment:
We should add `target="_blank"` when the link is external, so it opens in a new tab.